caught panic with bad ISP using DPI

My ISP is using DPI and blocks almost everything, including this site. Recently I got panic after adding `cs-lv` server to my server list:
<details>
<summary>panic</summary>

```
panic: interface conversion: dns.RR is *dns.A, not *dns.TXT

goroutine 82 [running]:
main.FetchCurrentDNSCryptCert(0xc000156018, 0xc0001792b0, 0x155f39f, 0x3, 0xc0005e21fa, 0x20, 0x3f, 0xc00024dc40, 0x13, 0xc00024dc60, ...)
	/Users/vbauer/go/src/github.com/DNSCrypt/dnscrypt-proxy/dnscrypt-proxy/dnscrypt_certs.go:49 +0x1928
main.(*ServersInfo).fetchDNSCryptServerInfo(0xc0001560a0, 0xc000156018, 0xc0005810ed, 0x5, 0xc00024dc40, 0x13, 0xc0005e21fa, 0x20, 0x3f, 0x0, ...)
	/Users/vbauer/go/src/github.com/DNSCrypt/dnscrypt-proxy/dnscrypt-proxy/serversInfo.go:251 +0x100
main.(*ServersInfo).fetchServerInfo(0xc0001560a0, 0xc000156018, 0xc0005810ed, 0x5, 0xc00024dc40, 0x13, 0xc0005e21fa, 0x20, 0x3f, 0x0, ...)
	/Users/vbauer/go/src/github.com/DNSCrypt/dnscrypt-proxy/dnscrypt-proxy/serversInfo.go:235 +0x220
main.(*ServersInfo).refreshServer(0xc0001560a0, 0xc000156018, 0xc0005810ed, 0x5, 0xc00024dc40, 0x13, 0xc0005e21fa, 0x20, 0x3f, 0x0, ...)
	/Users/vbauer/go/src/github.com/DNSCrypt/dnscrypt-proxy/dnscrypt-proxy/serversInfo.go:102 +0x1c9
main.(*ServersInfo).refresh(0xc0001560a0, 0xc000156018, 0x0, 0x0, 0xc0004d20f0)
	/Users/vbauer/go/src/github.com/DNSCrypt/dnscrypt-proxy/dnscrypt-proxy/serversInfo.go:137 +0x1b3
main.(*Proxy).StartProxy(0xc000156018)
	/Users/vbauer/go/src/github.com/DNSCrypt/dnscrypt-proxy/dnscrypt-proxy/proxy.go:158 +0x804
main.(*App).AppMain(0xc000156000, 0xc000156018)
	/Users/vbauer/go/src/github.com/DNSCrypt/dnscrypt-proxy/dnscrypt-proxy/main.go:106 +0x34
main.(*App).Start.func1(0xc000156000, 0xc000156018)
	/Users/vbauer/go/src/github.com/DNSCrypt/dnscrypt-proxy/dnscrypt-proxy/main.go:96 +0x35
created by main.(*App).Start
	/Users/vbauer/go/src/github.com/DNSCrypt/dnscrypt-proxy/dnscrypt-proxy/main.go:95 +0x123
```
</details>

I suspect that my ISP is tampering something, so there is `A` record received instead of expected `TXT`. I got following line with `spew.Dump`, while I was troubleshooting.
```
(*dns.A)(0xc0002d6000)(2.dnscrypt-cert.cryptostorm.is.	300	IN	A	127.0.0.1)
```
There is no such panic when I turn on vpn.

--- a/dnscrypt-proxy/dnscrypt_certs.go
+++ b/dnscrypt-proxy/dnscrypt_certs.go
@@ -43,7 +43,14 @@ func FetchCurrentDNSCryptCert(proxy *Proxy, serverName *string, proto string, pk
 	highestSerial := uint32(0)
 	var certCountStr string
 	for _, answerRr := range in.Answer {
-		binCert, err := packTxtString(strings.Join(answerRr.(*dns.TXT).Txt, ""))
+		var txt string
+		if t, ok := answerRr.(*dns.TXT); !ok {
+			dlog.Warnf("[%v] Certificate not found", providerName)
+			continue
+		} else {
+			txt = strings.Join(t.Txt, "")
+		}
+		binCert, err := packTxtString(txt)
 		if err != nil {
 			dlog.Warnf("[%v] Unable to unpack the certificate", providerName)
 			continue

